eq/client/compositor.cpp

```diff
@@ -1257,17 +1266,17 @@ void Compositor::_drawPixels( const Image* image, const ImageOp& op,
     texture->applyWrap();
 
     if ( which == Frame::BUFFER_COLOR )
-        glDepthMask( false );
+        EQ_GL_CALL( glDepthMask( false ))
```

Comment:
; is already part of EQ_GL_CALL. If done here, parentheses for new scope are needed. Looks awkward, but compiles